Description: Regexen for macros to create links in html documentation
Author: Daniel Bergey <bergey@alum.mit.edu>
Forwarded: Yes
Last-Update: 2011-02-01

--- a/Documentation/asciidoc.conf
+++ b/Documentation/asciidoc.conf
@@ -100,3 +100,9 @@
 [linkstgsub-inlinemacro]
 <a href="stg-{target}.html">{target}</a>
 endif::backend-xhtml11[]
+
+[macros]
+# regexen to match macro templates above
+(?P<name>linkman):(?P<target>\w+)\[(?P<attrlist>\d+)]=
+(?P<name>linkstgsub):(?P<target>\w+)\[]=
+(?P<name>linkstg):(?P<target>\w+)\[]=
